hugegraph-api/src/main/java/com/baidu/hugegraph/config/ServerOptions.java

```diff
@@ -63,7 +63,7 @@ public static synchronized ServerOptions instance() {
                     "will be rejected when the available memory of system " +
                     "is lower than this value.",
                     positiveInt(),
-                    128L
+                    512L
```

Comment:
if the program is allocated with 512M memory, in general, the program itself occupies part of the memory, so it may not be unable to serve.

Reply:
seems 256M is better